server/src/test/kotlin/fi/vauhtijuoksu/vauhtijuoksuapi/server/StreamMetadataTest.kt

```diff
@@ -114,13 +149,29 @@ class StreamMetadataTest : ServerTestBase() {
                     assertEquals(200, res.statusCode())
                     assertEquals("application/json", res.getHeader("content-type"))
                     someMetadataJson.put("donation_goal", 1000)
-                    assertEquals(someMetadataJson, res.bodyAsJsonObject())
+                    assertEquals(someMetadataJson.toString(), res.bodyAsJsonObject().toString())
```

Comment:
Why comparing strings? I think you'd get better error descriptions with json objects

Reply:
because identical json objects are recognized as different and I cannot tell why that is

Reply:
At least UUID has caused headaches for me in the past. It shows the same, but because it's UUID in one and String in the other `equals` says it's not the same

Reply:
now comparing api objects. Works better